contracts/Oracles/FutarchyOracle.sol

```diff
@@ -63,15 +64,17 @@ contract FutarchyOracle is Oracle {
         uint8 outcomeCount,
         int lowerBound,
         int upperBound,
-        MarketFactory marketFactory,
+        StandardMarketWithPriceLoggerFactory marketFactory,
         MarketMaker marketMaker,
         uint24 fee,
-        uint _deadline
+        uint _deadline,
+        uint startDate
+
     )
         public
     {
         // Deadline is in the future
-        require(_deadline > now);
+        require(_deadline > 0);
```

Comment:
Ah, deadline typically refers to an absolute point in time. I recommend either renaming to trading period or restoring the original logic.